.github/workflows/tag.yml

```diff
@@ -0,0 +1,40 @@
+name: Pre-release Tag
+
+on:
+  pull_request_target:
+    types: [closed]
+    branches: [master]
```

Comment:
Pull request target and is safe here because we don't build any code so there no chance secrets could be leaked to a 3rd party. We only create a tag once PRs are merged